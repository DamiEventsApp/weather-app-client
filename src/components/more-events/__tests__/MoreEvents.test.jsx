import React from 'react';
import { shallow } from 'enzyme';
import MoreEvents from '../MoreEvents.component.jsx';
import '../../../../setupTests.js';

describe('MoreEvents', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<MoreEvents />);
        expect(wrapper).toMatchSnapshot();
        // On the first run of this test, Jest will generate a snapshot file automatically.
    });
});