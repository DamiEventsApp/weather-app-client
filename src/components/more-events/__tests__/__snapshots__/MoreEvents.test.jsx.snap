// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoreEvents renders correctly 1`] = `ShallowWrapper {}`;
